index.html

```diff
@@ -6,15 +6,214 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" type="text/css" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <header class="container">
+        <section>
+            <div class="row">
+                <div class="col-3">
+                    <p class="name">Tam Bahri</p>
+                    <p class="job">Personal Trainer</p> 
+                </div>
+                <div class="row">
+                    <div class="col-3 md-col-8">
+                        <nav>
+                            <ul class="nav-list">
+                                <li>Home</li>
+                                <li>About</li>
+                                <li>Services</li>
+                                <li>Motivation</li>
+                                <li>Classes</li>
+                                <li>Food and Health</li>
+                                <li>Blog</li>
+                                <li>Contact</li>  
+                            </ul>
+                        </nav>
+
+                    </div>
+                   
+
+                </div>
+                
+                
+                <div>
+                    <i class=" fa fa-user-circle-o"></i>
+                    
+                    <p class="login">Login</p>
+                </div>
+                
+            </div>
+
+        </section>
+       
+      
+          
+    </header>
+
+    <section class="container">
```

Comment:
Consider moving everything down one level so that you:
- first set your background image (without a `container` around it so it goes all the way to the edges of the screen). This means moving your `hero` class to the top; 
- then, you add the `container` _inside_ of that (as a child of the element that has the background image)
- then, `row`
- then `col-12`

So basically the same as you've done except you would be moving the `hero` class to wrap everything else. This will have a much better visual effect 😄